File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/editor/TilePlacedMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.editor;


import de.hsrm.mi.swtpro03.server.game.dtos.TileDTO;

public class TilePlacedMessage {

	public static final String TYPE = "TilePlacedMessage";

	private TileDTO tileDTO;

	public TilePlacedMessage() {
	}

	public TilePlacedMessage(TileDTO tileDTO) {
		this.tileDTO = tileDTO;
	}

	public TileDTO getTileDTO() {
		return tileDTO;
	}

	public void setTileDTO(TileDTO tileDTO) {
		this.tileDTO = tileDTO;
	}
}

File: client/Client/Network/Messages/Editor/TilePlacedMessage.cs
using LoadRunnerClient.DTOs;

namespace LoadRunnerClient.Messages
{
    /// Author Florian Ortmann
    public class TilePlacedMessage
    {
        public const string TYPE = "TilePlacedMessage";

        public TilePlacedMessage()
        {
        }

        public TilePlacedMessage(TileDTO tileDTO)
        {
            this.tileDTO = tileDTO;
        }

        private TileDTO _tileDTO;

        public TileDTO tileDTO
        {
            get => _tileDTO;
            set => _tileDTO = value;
        }
    }
}